docs/api-docs/getting-started/webhooks/webhook-events.md

```diff
@@ -59,7 +39,289 @@
 }
 ```
 
-<a id="webhook-events_orders"></a>
+| Name | Definition |
+| -- | -- |
+| `scope` | The [event](/api-docs/getting-started/webhooks/webhook-events) registered when the webhook was created. |
+| `store_id` | A numerical identifier that is unique to each store. |
+| `data` | A lightweight description of the [event](/api-docs/getting-started/webhooks/webhook-events) that triggered the webhook. Will vary depending on the event registered. |
+| `hash` | The payload data json encoded then passed through SH1 encryption. |
+| `created_at` | Unix timestamp of the date the hook was created. |
+| `producer` | Will always follow the pattern `stores/store_hash`. This is the store that created the webhook. |
+
+
+
+## Cart
+
+```json
+{
+    "scope": "store/cart/created",
+    "store_id": "1025646",
+    "data": {
+        "type": "cart",
+        "id": "09346904-4175-44fd-be53-f7e598531b6c"
+    },
+    "hash": "352e4afc6dd3fc85ea26bfdf3f91852604d57528",
+    "created_at": 1561482670,
+    "producer": "stores/{store_hash}"
+}
+```
+
+| Name | Description |
+|-|-|
+| store/cart/* | Subscribe to all cart events. This will also subscribe you to cart/lineItem. |
+| store/cart/created | This webhook will fire whenever a new cart is created, either via a storefront shopper adding their first item to the cart, or when a new cart is created via an API consumer. If it is from the storefront, then it fires when the first product is added to a new session.(The cart did not exist before). For the API it means a `POST` to /carts, (V3 and Storefront API). The `store/cart/updated` hook will also fire.|

```

Comment:
```suggestion
| store/cart/created | This webhook will fire whenever a new cart is created, either via a storefront shopper adding their first item to the cart, or when a new cart is created via an API consumer. If it is from the storefront, then it fires when the first product is added to a new session. (The cart did not exist before.) For the API it means sending a `POST` request to `/carts`, (V3 and Storefront API). The `store/cart/updated` hook will also fire.|

```
This explanation is a bit hard to follow. Maybe simplify or shorten the sentences to make them easier to comprehend. For example:

This webhook will fire whenever a new cart is created either though a storefront or by sending a `POST` request to `/carts` (V3 and Storefront APIs). For carts created via storefront, the webhook will fire when the first product is added to a new session. (The cart did not exist before.) When you create a new cart using an API consumer, the `store/cart/updated` hook will fire in addition to `store/cart/created`.